### PR TITLE
Import AC97 driver sample

### PR DESCRIPTION
**Pull request by Extravert-ir**
Reopening #4246. Also it makes sense to reference the "original" PR #3141

### CODE REVIEW

**Comment by Vayne2500** [comment]
By the way, this is needed to be made into another PR:

https://github.com/reactos/reactos/pull/4305

Without it, it will crash the OS when booting.

**Comment by binarymaster** [comment]
> Also it makes sense to reference the "original" PR #3141

I think the real origin of this code is this branch: https://github.com/TheDeadFish/reactos/commits/ac97_import_new2

File: media/doc/3rd Party Files.txt [hunk 1]
```diff
@@ -151,6 +151,12 @@ Used Version: ???
 License: GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later.html)
 URL: http://www.vmware.com/download/open_sources.html
 
+Title: AC97 Sample Driver
+Path: drivers/wdm/audio/drivers/AC97
```
Comment (binarymaster) [inline media/doc/3rd Party Files.txt:155]:
```suggestion
Path: drivers/wdm/audio/drivers/ac97
```

Comment (JoachimHenze) [reply media/doc/3rd Party Files.txt:155]: I am no fan of frequent restructuring in the source tree.
Atm this folder does contain 3 subdirs: cmiDriver, mpu401, sb16. All of them are sound related. Fine.
I don't see any real benefit to move them around. I mean we are within root/drivers already. It's not like that they would be heavily misplaced within root/dll or something.

Comment (Extravert-ir) [reply media/doc/3rd Party Files.txt:155]: Audio folders are a mess, `wdm` is a useless folder there. It needs more than just a renaming

**Comment by Extravert-ir** [comment]
This PR waits for portcls driver fixes to be merged.

**Comment by JoachimHenze** [comment]
Attention: This PR did unhide:
https://jira.reactos.org/browse/CORE-18678

**Pull request merged**